### PR TITLE
RSDK-7021 Update motion algorithms page

### PR DESCRIPTION
**Pull request by raybjork**
We have added TP-space RRT to our list of offerings and therefore should update our docs on this subject.

### CODE REVIEW

File: docs/mobility/motion/algorithms.md
Comment (raybjork) [inline docs/mobility/motion/algorithms.md:38]: @biotinker is this section still accurate?  Should it be improved?

File: docs/mobility/motion/algorithms.md
Comment (npentrel) [inline docs/mobility/motion/algorithms.md:55]:
```suggestion
This algorithm is based on the research described in the paper: [TP-Space RRT – Kinematic Path Planning of Non-Holonomic Any-Shape Vehicles](https://www.researchgate.net/publication/275584014_TP-Space_RRT_-_Kinematic_Path_Planning_of_Non-Holonomic_Any-Shape_Vehicles). This algorithm is suitable to planning for Bases since it is able to incorporate non-holonomic constraints into the planner directly to only return plans that are kinematically feasible for the base.  One such example of a constraint that this allows us to impose is a minimum turning radius constraint, which is expressed through the [Base's](/components/base/) [GetProperties](https://docs.viam.com/components/base/#getproperties) method.  
```

File: docs/mobility/motion/algorithms.md
Comment (npentrel) [inline docs/mobility/motion/algorithms.md:57]:
```suggestion
This algorithm works by exploring a series of continuous trajectory-parameter spaces (TP-space) which describe different actions the base can take, such as, for example, turn and then go straight.
This concept is then used within an RRT framework to compose these different actions into a series of sequential actions to be taken in order to navigate the base to its goal.  
```

File: docs/mobility/motion/algorithms.md
Comment (npentrel) [inline docs/mobility/motion/algorithms.md:55]:
```suggestion
This algorithm is based on the research described in the paper: [TP-Space RRT – Kinematic Path Planning of Non-Holonomic Any-Shape Vehicles](https://www.researchgate.net/publication/275584014_TP-Space_RRT_-_Kinematic_Path_Planning_of_Non-Holonomic_Any-Shape_Vehicles).
This algorithm is suitable to planning for bases since it is able to incorporate non-holonomic constraints into the planner directly to only return plans that are kinematically feasible for the base.
One such example of a constraint that this allows us to impose is a minimum turning radius constraint, which is expressed through the [Base's](/components/base/) [GetProperties](https://docs.viam.com/components/base/#getproperties) method.  
```

**Review by npentrel — APPROVED** [review]
LGTM, once @biotinker approves

**Comment by npentrel** [comment]
It does also look like you need to rebase

File: docs/mobility/motion/algorithms.md
Comment (biotinker) [reply docs/mobility/motion/algorithms.md:38]: This is still accurate. We haven't changed arm stuff.

File: docs/mobility/motion/algorithms.md
Comment (biotinker) [inline docs/mobility/motion/algorithms.md:53]: TP-Space RRT

File: docs/mobility/motion/algorithms.md
Comment (biotinker) [inline docs/mobility/motion/algorithms.md:58]: Not for this PR perhaps, but we may want to mention course correction somewhere

**Review by biotinker — APPROVED** [review]
LGTM, one non-blocking update to remove a hyphen

**Pull request merged**
